readme.md

```diff
@@ -1,38 +1,49 @@
 # filter mod
 
-This mod adds a simple chat filter. There is no default word list,
-and adding words to the filter list is done through the `/filter`
-chat command. You need the `server` priv to use the chat command.
+This mod adds a chat filter API that can be used to register custom
+chat filters. As of now a bad words filter has been implemented. There
+is no default word list, and adding words to the filter list is done
+through the `/word_filter` chat command, which requires `server` priv.
 
-The `/filter` chat command can `add`, `remove` or `list` words. The
+The `/word_filter` chat command can `add`, `remove` or `list` words. The
```

Comment:
don't like this.

perhaps, `/filter words add <word>` would be more sensical, then we can add more stuff later using the same format of the commands:

`/filter <filtername> <command> <value>`

Reply:
Good idea

Reply:
But doesn't that result in too many params? I was thinking of separate `word_filter`, `spam_filter`, `caps_filter`, etc.

Reply:
too many /commands is just as bad, or even worse, since alphabetical sorting will make them hard to find if you do `/help`. So, adding subcommands is probably the way to go here.

Reply:
@sofar Is this the only thing that needs to be addressed?